docs/guides/auth.md

```diff
@@ -175,26 +175,41 @@ var scope = 'https://www.googleapis.com/auth/grpc-testing';
 
 ```csharp
 // Base case - No encryption/authentication
```

Comment:
Yeah, I'm happy to merge this now and we can make further improvements in a future PR. Who is the person to talk to for the more "generic" Authenticating With Google paragraph above?